app/src/main/java/camp/nextstep/edu/calculator/MainActivity.kt

```diff
@@ -1,26 +1,63 @@
 package camp.nextstep.edu.calculator
 
 import android.os.Bundle
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import camp.nextstep.edu.calculator.databinding.ActivityMainBinding
+import com.nextstep.edu.domain.CalculationExpression
+import com.nextstep.edu.domain.Calculator
+import com.nextstep.edu.domain.Operator
 
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
-
+    private var calculationExpression: CalculationExpression = CalculationExpression()
```

Comment:
```suggestion
    private val calculationExpression: CalculationExpression = CalculationExpression()
```

혹시 이 부분 var로 선언한 이유가 있을까요?
해당 객체를 다시 선언하거나 하는 등의 로직이 없는 것 같아서 질문드려요~
크게 이유가 없다면 불변타입인 val을 이용해주는 것이 좋습니다 :)